Formula/p/parallel.rb

```diff
@@ -17,6 +17,8 @@ class Parallel < Formula
     sha256 cellar: :any_skip_relocation, all: "e272a8250ab0451b6bc1891228dd6d7577d5532d6c4cb409a3e3551e44395664"
   end
 
+  uses_from_macos "perl"
```

Comment:
Parallel doesn't really require a specific version of `perl`, which is what we use `uses_from_macos "perl"` for.

Unless you're having problems with using `parallel` without this?

Reply:
How about Linux? Or any system where there's no pre-installed perl?